### DISCUSSION TITLE
„Goto File“ in „Browse Code“

### DISCUSSION THREAD

**Discussion by staabm** [community/community#4704] [Mobile] [answered]
I would love to have the "goto file" feature available in the "browse code" in a repo section within the github mobile app.

It could be accessible e.g. via the "..." menu

**Comment by brianlovin** [comment, marked as answer]
Great idea, thanks for the note @staabm – we have a long-running list of improvements we're hoping to make on the Browse Code area of the apps in a future update, and this will definitely be on there.
